helm/templates/prometheus-rules.yaml

```
@@ -0,0 +1,43 @@
+{{- if and .Values.exporter.serviceMonitor.enabled .Values.exporter.serviceMonitor.prometheusRule.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "cvms.labels" . | nindent 4 }}
+    {{- include "cvms.exporterSelectorMetricLabels" . | nindent 4 }}
+    {{- with .Values.exporter.metrics.labels }}
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)